### ISSUE TITLE
Add code of conduct to all NF GitHub repos

### ISSUE THREAD

**Issue by logankilpatrick** [numfocus/.github#1]
We should create a CoC file in this repo with the NF code of conduct, cc @arliss-NF

**Comment by arliss-NF** [comment]
Yes @logankilpatrick - that sounds like a good idea.  Will look into it ...

**Comment by Student-ShivamChauhan** [comment]
hello @logankilpatrick i am interested to contribute in this issue please assign me this issue

**Comment by sanchalitorpe13** [comment]
Hi @logankilpatrick,  
I’d love to contribute to this issue by adding a Code of Conduct file to the repository, following the NumFOCUS CoC.  
Please assign this issue to me if possible 🙂
